src/odb/include/odb/dbStream.h

```diff
@@ -381,6 +400,7 @@ class dbIStream
   template <size_t I = 0, typename... Ts>
   constexpr dbIStream& operator>>(std::tuple<Ts...>& tup)
   {
+    static_assert(I <= kTemplateRecursionLimit, "OpenROAD disallows of std::variants larger than 16 elements. You should look into alternate solutions");
```

Comment:
Probably should be std::tuple in this message.

Reply:
Thanks!, good catch!